From c646b3b411c9bd8c0b4d7116b925ab9bb98c1971 Mon Sep 17 00:00:00 2001
From: Stefan Roese <sr@denx.de>
Date: Tue, 18 Dec 2018 09:26:32 +0100
Subject: [PATCH] MIPS: ralink: dts: gardena_smart_gateway_mt7688: Add
 bootcounter DT node

Add the U-Boot bootcounter DT node, so that the driver can register on
it.

Signed-off-by: Stefan Roese <sr@denx.de>
---
 arch/mips/boot/dts/ralink/gardena_smart_gateway_mt7688.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/mips/boot/dts/ralink/gardena_smart_gateway_mt7688.dts b/arch/mips/boot/dts/ralink/gardena_smart_gateway_mt7688.dts
index b6cd5d23846e..963f79fc6a08 100644
--- a/arch/mips/boot/dts/ralink/gardena_smart_gateway_mt7688.dts
+++ b/arch/mips/boot/dts/ralink/gardena_smart_gateway_mt7688.dts
@@ -162,3 +162,10 @@
 &wmac {
 	status = "okay";
 };
+
+&palmbus {
+	bootcount@0x6c {
+		compatible = "uboot,bootcount";
+		reg = <0x0000006c 0x4>;
+	};
+};
